scripts/globals/spells/invisible.lua

```diff
@@ -15,19 +16,14 @@ end
 function onSpellCast(caster, target, spell)
     if not target:hasStatusEffect(dsp.effect.INVISIBLE) then
 
-        -- last 7-9 minutes
-        local duration = calculateDuration(math.random(420, 540), spell:getSkillType(), spell:getSpellGroup(), caster, target)
-        duration = calculateDurationForLvl(duration, 25, target:getMainLvl())
+        local duration = calculateDuration(math.random(42, 54) * 10, spell:getSkillType(), spell:getSpellGroup(), caster, target)
 
-        -- FIXME: Create mod and use that instead of an itemID check
-        if (target:getEquipID(dsp.slot.BACK) == 13692) then -- skulker's cape
-            duration = duration * 1.5
-        end
+        duration = calculateDurationForLvl(duration, 25, target:getMainLvl()) + player:getMod(dsp.mod.INVISIBLE_DURATION)
```

Comment:
This should only increase the duration on the wearer. Also, it should be added into the base duration, not tacked on after a level check (won't matter because you can't cast it on yourself if you're level synced below the spell level)